### ISSUE TITLE
Installing the Netlify CLI has recently started failing because Node 10.22 is no longer supported

### ISSUE THREAD

**Issue by Haegin** [netlify/cli#1616]
**Describe the bug**

Our CI system installs the Netlify CLI to deploy after a build. This has been working well for some time but yesterday it started failing to install with the following error:

```
error @netlify/plugins-list@2.0.0: The engine "node" is incompatible with this module. Expected version ">=12". Got "10.22.0"00:20
error Found incompatible module.
```

**To Reproduce**

1. Install and use NodeJS 10.22
2. Install the CLI, pinned to a version that was previously working correctly. `yarn global add "netlify-cli@2.69.0"`

**Configuration**

N/A

**Expected behavior**

I would expect the Netlify CLI to install correctly - dropping support for NodeJS v10 is definitely a breaking change and dependencies should be pinned to ensure new releases aren't automatically pulled in.

**Comment by ehmicky** [comment]
Hi @Haegin,

Thanks for posting this!
Node.js 10.22 is supported. 
The problem was due to one of our dependencies which was just added and advertised supporting only Node 12, while it actually support any Node version (the npm package just exports a JSON file).

This should be fixed by https://github.com/netlify/plugins/pull/183

**Comment by erezrokah** [comment]
Reopening until we publish a new version

**Comment by Haegin** [comment]
Ah that's great to hear. Thanks for the speedy response. Can you give any rough estimate on when a new version will be published?

**Comment by erezrokah** [comment]
Hi @Haegin, we've released [v2.69.1](https://github.com/netlify/cli/releases/tag/v2.69.1) to `npm` with a fix to this issue.
Closing the issue - please comment if the problem still happens with `v2.69.1`.
